# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======

GetGraspingPointActionGoal action_goal
GetGraspingPointActionResult action_result
GetGraspingPointActionFeedback action_feedback
